contrib/storage-jdbc/src/test/resources/mysql-test-data.sql

```diff
@@ -0,0 +1,53 @@
+
+set global time_zone = "+00:00";
```

Comment:
same question here as above. Any chance we can factor out the differences and keep the rest shared?